py/torch_tensorrt/dynamo/_compiler.py

```diff
@@ -295,8 +332,19 @@ def compile_module(
         submodule = getattr(partitioned_module, name)
         # Criteria for a module to be convertible to TRT
         if settings.use_fast_partitioner and "_run_on_acc" not in name:
+            dryrun_tracker.to_run_in_torch.extend(parse_non_trt_nodes(submodule))
```

Comment:
- In `global_partitioning` - `named_children` only returns TRT modules'
- Can add feature for `fast_partitioning` first, then expand to global later?